app/models/concerns/cable_ready/updatable.rb

```diff
@@ -30,11 +50,14 @@ def self.enable_cable_ready_updates(*options)
           options = options.extract_options!
           options = {
             on: [:create, :update, :destroy],
-            if: -> { true }
+            if: -> { true },
+            debounce: 0.seconds
```

Comment:
I wonder if it would make sense to have the default provide some degree of protection. Perhaps 100 milliseconds?

Reply:
well I switch on the debounce time in `broadcast_updates` to gracefully allow the default behavior. But sure, why not!

Reply:
actually I just noticed that this breaks all the tests, so we'd have to specify `debounce: 0` everywhere. I'd recommend merging this PR and then introducing this in a separate one to keep things readable